InformedSolutions/JAQU-CAZ-National-Taxi-Register-UI
# [NTR-483] Add SES configuration

## Motivation and Context


https://eaflood.atlassian.net/browse/NTR-482
https://eaflood.atlassian.net/browse/NTR-483

## Description

I added [aws-sdk-rails gem](https://github.com/aws/aws-sdk-rails) to handle sending emails and a basic mailer.

## How Has This Been Tested?




1. Add `SES_ACCESS_KEY_ID` and `SES_SECRET_ACCESS_KEY` env variables
2. Go to AWS Console -> SES -> Email Addresses -> Verify a new email address
3. Add your email and click the link that will be sent to you
4. Set `SES_FROM_EMAIL` with some verified email
5. Run this in `rails console`
```
user = User.new
user.email = <your verified email>
UploadMailer.success_upload(user).deliver
```
6. You should receive en email

## Screenshots (if appropriate):

![Screen Shot 2019-09-18 at 11 09 48](https://user-images.githubusercontent.com/16211686/65134641-e8162300-da04-11e9-8b94-9abc32bce519.png)

## Checklist:


- [x] Includes a link to an issue (if apply)
